DQMOffline/L1Trigger/python/L1TriggerDqmOfflineMC_cff.py

```diff
@@ -22,11 +22,14 @@
 
 
 # do not run the emulator client in MC
-l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient) 
+l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient)
 
+import DQMOffline.L1Trigger.L1TriggerDqmOffline_cff as stage2_cffs_
+stage2_cffs_.Stage2l1TriggerDqmOffline.remove(stage2_cffs_.l1tStage2EmulatorMonitorClient)
```

Comment:
Hi @cbbrainerd @rekovic - sorry for my slow review - there are a few cfg issues in this PR all revolving around the same theme - that is that objects should get manipulated where they are declared. So, could Stage2l1TriggerDqmOffline be modified directly in L1TriggerDqmOffline_cff?